app/src/main/java/com/github/sdpteam15/polyevents/adapter/ItemAdapter.kt

```diff
@@ -33,28 +32,29 @@ class ItemAdapter(
      * adapted ViewHolder for each item
      * Takes the corresponding event view
      */
-    inner class ItemViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
+    inner class ItemViewHolder(view: View) : RecyclerView.ViewHolder(view) {
 
+        // TODO: Consider adding field for itemType
         private val itemName = view.findViewById<TextView>(R.id.id_list_item_name)
         private val btnRemove = view.findViewById<ImageButton>(R.id.id_remove_item)
 
         /**
          * Binds the values of each view of an event to the layout of an event
          */
-        fun bind(item: String) {
+        fun bind(item: Item) {
             btnRemove.setOnClickListener {
-                Database.currentDatabase.removeItem(item)
+                currentDatabase.removeItem(item)
                 // TODO ADD data.postValue(items)
                 notifyDataSetChanged()
             }
-            itemName.text = item
+            itemName.text = item.itemType.itemType + item.itemId
```

Comment:
Is it really necessary to add the itemId in the item name ? I don't think so :)

Reply:
We don't "have to", but I think it's better than having multiple items with the same name, it's just a way to differentiate them. What do you think?

Reply:
The thing is that ItemAdapter will display each item individually if I understood correctly, and not group items with the same itemType and put a number beside to denote how many we have. 

Again, there's still some work to be done on Items to clearly define how are we going to work with them.